src/ProgressOnderwijsUtils/Data/FieldMapping.cs

```diff
@@ -55,6 +55,7 @@ public struct FieldMappingValidation
     public bool AllowExtraSourceColumns;
     public bool AllowExtraTargetColumns;
     public bool OverwriteAutoIncrement;
+    public bool AllowReadOnlyTarget;
```

Comment:
```suggestion
    public bool SilentlySkipReadonlyTargetColumns;
```

want "allow" wekt een beetje de illusie dat we er iets mee doen, maar we negeren ze gewoon.